Tests/SwiftyJSONTests/NestedTests.swift

```diff
@@ -0,0 +1,53 @@
+//
```

Comment:
There is an existed file call `NestedJSONTests.swift`. Would you mind moving your test into that file?